manifests/example-config.yaml

```diff
@@ -43,6 +43,10 @@ data:
       # smurf protection. Such devices have become fairly rare, but
       # the option is here if you encounter serving issues.
       avoid-buggy-ips: true
+      # (optional) If false, MetalLB will not automatically allocate
```

Comment:
Can you say `(optional, default true)`, to make it clear that you have to disable it?